db/schema.rb

```diff
@@ -37,7 +37,7 @@
   create_table "builds", id: :integer do |t|
     t.integer "project_id", null: false
     t.integer "number"
-    t.string "git_sha", null: false
+    t.string "git_sha", limit: 128, null: false
```

Comment:
I don't think we want all of these changes...

Reply:
Other than the migration for use env repo I didn’t make any changes to the repository.

Reply:
change it back manually or `rake db:schema:load` from master and then run migration again

Reply:
hmm nvm looks like 128 is the correct value from db/migrate/20150508140514_create_builds.rb

Reply:
idk ... still this should be it's own PR